src/runtime/pythonengine.cs

```diff
@@ -742,9 +707,10 @@ public static KeywordArguments kw(params object[] kv)
             return dict;
         }
 
+        [Obsolete("Use PyModule.Import")]
         public static PyObject Import(string name)
```

Comment:
I wouldn't obsolete this one right now. Having `Py.` as a central entry-point has merit.

Reply:
Perhaps we should move `Py` class into a new file
![image](https://user-images.githubusercontent.com/239520/109041621-167d4080-7684-11eb-82ec-a7747c9e268a.png)

Reply:
:D
Yep, sorry for the inconsistency. I'd like to obsolete `PythonEngine` for the most part and collect entry points in `Py`.

Reply:
Removed this `[Obsolete]`, added a new issue to move `Py` out.